content/use-cases/use-an-escrow-as-a-smart-contract.md

```diff
@@ -0,0 +1,137 @@
+
+- Access to a secure signing environment, which includes having a network connection to a [`rippled` server](install-rippled.html) (any server) that they can submit signed transactions to. <!--#{ once set up secure signing tutorial is available, link to it from here }# -->
+
+The party band (receiver) must have:
+
+- An XRP Ledger [account](accounts.html#creating-accounts) that can receive the XRP paid by the escrow.
+
+- Access to a [`rippled` server](install-rippled.html) that they can use to look up the details of an XRP Ledger transaction hash and submit the fulfillment value to finish the escrow.
+
+The party planner (oracle) must have:
+
+- The ability to generate a condition and a fulfillment.
+
+- To be able to keep a secret (the fulfillment) until the time is right.
+
+- A way to communicate the fulfillment publicly or at least to the party band when the time is right.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## Define the terms of the smart contract
+
+To create the escrow as a smart contract, the participants must first define the terms of the contract. In this scenario, the participants need to agree on the following details.
+
+- **Should the escrow disallow fulfillment until a specific time?**
+
+      While this is an option, the participants agree that it is unnecessary for their escrow. For conditionally-held escrows, enabling this option doesn't provide any additional security, since whether the escrow can be finished still depends entirely on whether the party planner (oracle) publishes the fulfillment before the expiration.
+
+- **Should the escrow expire?**
+
+      Absolutely yes. The participants agree that the escrow should expire after 12 noon the day after the party. This gives the party band (receiver) enough time to finish the escrow, after the party planner verifies that they fulfilled their end of the contract and publishes the cryptographic fulfillment. After expiration, the locked XRP returns to the party host's (sender's) account.
+
+      If the participants don't allow the escrow to expire and the party planner doesn't release the condition, the XRP stays locked in the escrow forever.
+
+- **How much XRP should the escrow lock up and potentially pay?**
+
+      The participants agree that the escrow should lock up and potentially pay 2000 XRP, which is the party band's fee.
+
+- **From which XRP Ledger account should the escrow lock up XRP for potential payment to the party band?**
+
+      The participants agree that the escrow should lock up and potentially pay XRP out of the party host's XRP Ledger account.
+
+- **Which XRP Ledger account should the escrow potentially pay XRP to?**
+
+      The participants agree that the escrow should potentially pay XRP to the party band's XRP Ledger account.
+
+
+
+<span class="use-case-step-num">{{n.next()}}</span>
+## [Generate a condition and a fulfillment](send-a-conditionally-held-escrow.html#1-generate-condition-and-fulfillment) (oracle)
```

Comment:
Similar to the Payment Channels case, I like to structure these sorts of headings as "Role: task" rather than "task (role)" because I prefer to set the context first.